core/src/main/java/io/aiven/klaw/service/EnvsClustersTenantsControllerService.java

```diff
@@ -555,6 +555,35 @@ public List<EnvModel> getSchemaRegEnvs() {
     return envModelList;
   }
 
+  public List<EnvModel> getRequestSchemaEnvs() {
+    int tenantId = getUserDetails(getUserName()).getTenantId();
+    String orderOfEnvs = mailService.getEnvProperty(tenantId, "ORDER_OF_SCHEMA_ENVS");
+    String requestSchemasEnvs = mailService.getEnvProperty(tenantId, "REQUEST_SCHEMA_OF_ENVS");
+
+    String[] reqSchemaEnvs = requestSchemasEnvs.split(",");
```

Comment:
npe possible ?

Reply:
GetEnvProperty always returns a "" if any issue occurs so no npe currently possible.

Reply:
If tenant config does not define ORDER_OF_SCHEMA_ENVS or for topics, by default it is null I think. 
Are you sure it returns "", as I see npe coming from this block.